### DISCUSSION TITLE
workflow does not work under   Execution schedule (defined time intervals)

### DISCUSSION THREAD

**Discussion by newminder** [YetiForceCompany/YetiForceCRM#17111] [General] [answered]
I tried every 5 15 30 mins,  all would not work.  is this need to be paid?

**Comment by rskrzypczak** [comment, marked as answer]
> I tried every 5 15 30 mins, all would not work.

it works.
check if your cron is running (https://doc.yetiforce.com/administrator-guides/automation/cron/)

> is this need to be paid?

no.

**Reply by newminder** [reply]
thank you very much, i created cron jobs according the doc you offered.  it works now.
